tests/aggregation/tests.py

```diff
@@ -18,6 +22,20 @@
 from .models import Author, Book, Publisher, Store
 
 
+class NowUTC(Now):
```

Comment:
This doesn't seem so friendly for third-party backends. CockroachDB needs `NowUTC.as_cockoachdb = NowUTC.as_postgresql` but I'd prefer not to put code dealing with Django's tests with [the rest of function patching](https://github.com/cockroachdb/django-cockroachdb/blob/master/django_cockroachdb/functions.py). Thoughts?

Reply:
We can add `DatabaseFeatures.test_now_utc_template`.

Reply:
Sorry @timgraham, didn't think about other backends.

I was wondering about adding an optional `utc` argument to `django.db.models.functions.Now()`? I think it could be useful.

It was out of scope for this ticket though, hence this custom function.